charts/athens-proxy/templates/_helpers.tpl

```diff
@@ -3,8 +3,8 @@
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 {{- define "livenessPath" -}}
-{{- if eq .Values.image.tag "v0.3.0" -}}/{{- else -}}/healthz{{- end -}}
+{{- if contains "v0.2.0" .Values.image.tag -}}/healthz{{- else -}}/{{- end -}}
 {{- end -}}
 {{- define "readinessPath" -}}
-{{- if eq .Values.image.tag "v0.3.0" -}}/{{- else -}}/readyz{{- end -}}
+{{- if contains "v0.2.0" .Values.image.tag -}}/readyz{{- else -}}/{{- end -}}
```

Comment:
@rimusz `v0.2.0` did not have the `/readyz` endpoint (but /healthz was there)
https://github.com/gomods/athens/blob/c6644f82f9f9cf583a12566d844d9f132d06daa0/cmd/proxy/actions/app_proxy.go#L22-L23
`v0.3.0` and up does have /healthz and /readyz enpoints
https://github.com/gomods/athens/blob/a4450c114235eb18093328f1755c5258b771787b/cmd/proxy/actions/app_proxy.go#L25-L26

Reply:
so the old one:
```
{{- if eq .Values.image.tag "v0.3.0" -}}/{{- else -}}/healthz{{- end -}}
{{- end -}}
{{- define "readinessPath" -}}
{{- if eq .Values.image.tag "v0.3.0" -}}/{{- else -}}/readyz{{- end -}}
{{- end -}}
```
is incorrect

Reply:
ideally the new chart version should only support `v0.3.x`

Reply:
updated per @marpio comments in the commit `hardcode /healthz to deployment.yaml`